desktop/apps/categories2/components/GeneFamilies.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
```

Comment:
Not really. It's best to keep components in their own files. So `GeneFamily` should be in its own component. But for now, while we're sketching this out, I think it's fine.